config/nextcloud-27/nextcloud-27-deprecations.php

```diff
@@ -2,19 +2,34 @@
 
 declare(strict_types=1);
 
+use Nextcloud\Rector\Rector\AnnotationToAttributeRector;
 use Nextcloud\Rector\Set\NextcloudSets;
 use Rector\Config\RectorConfig;
-use Rector\Php80\Rector\Class_\AnnotationToAttributeRector;
 use Rector\Php80\ValueObject\AnnotationToAttribute;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->sets([NextcloudSets::NEXTCLOUD_26]);
     $rectorConfig->ruleWithConfiguration(
         AnnotationToAttributeRector::class,
         [
+            new AnnotationToAttribute('AnonRateThrottle', 'OCP\AppFramework\Http\Attribute\AnonRateLimit'),
+            // This one is commented out because the parameter would need to be transformed into several attributes
+            // new AnnotationToAttribute(
+            //     'AuthorizedAdminSetting',
+            //     'OCP\AppFramework\Http\Attribute\AuthorizedAdminSetting',
+            // ),
+            new AnnotationToAttribute('BruteForceProtection', 'OCP\AppFramework\Http\Attribute\BruteForceProtection'),
+            new AnnotationToAttribute('CORS', 'OCP\AppFramework\Http\Attribute\CORS'),
             new AnnotationToAttribute('NoAdminRequired', 'OCP\AppFramework\Http\Attribute\NoAdminRequired'),
-            new AnnotationToAttribute('PublicPage', 'OCP\AppFramework\Http\Attribute\PublicPage'),
             new AnnotationToAttribute('NoCSRFRequired', 'OCP\AppFramework\Http\Attribute\NoCSRFRequired'),
+            new AnnotationToAttribute(
+                'PasswordConfirmationRequired',
+                'OCP\AppFramework\Http\Attribute\PasswordConfirmationRequired',
+            ),
```

Comment:
Can you also put this one on one line or if that is not allowed then put everything else onto multiple lines as well? This is a bit ugly :see_no_evil: